pulsar-broker/src/main/java/org/apache/pulsar/PulsarStandaloneBuilder.java

```diff
@@ -97,10 +108,31 @@ public PulsarStandaloneBuilder withAdvertisedAddress(String advertisedAddress) {
         return this;
     }
 
-    public PulsarStandalone build() {
-        ServiceConfiguration config = new ServiceConfiguration();
-        config.setClusterName("standalone");
-        pulsarStandalone.setConfig(config);
+    public PulsarStandalone build()
+    {
```

Comment:
Move opening braces to previous line to follow formatting conventions

Reply:
Done.